locopy/snowflake.py

```diff
@@ -197,7 +197,7 @@ def connect(self):
         if self.connection.get("schema") is not None:
             self.execute("USE SCHEMA {0}".format(self.connection["schema"]))
 
-    def upload_to_internal(self, local, stage, parallel=4, auto_compress=True):
+    def upload_to_internal(self, local, stage, parallel=4, auto_compress=True, overwrite=True):
```

Comment:
I definitely think having default `overwrite=True` is easier for us to use since it aligns better with the old behaviour of `PUT` for snowflake connector, but not sure if we should set default to `False` to align with new behaviour so it doesn't cause confusion in the long run.

Reply:
Personally I like overwrite being `True` by default but that is just me. It is documented which should make the difference obvious, but totally a valid point! Anyone else have opinions? @theianrobertson @jborchma @dasjyao

Reply:
Hmm, `True` would be consistent with the older versions of the library. So maybe we can keep that for consistency.

Reply:
:+1: @elzzhu Are you OK with this?